### ISSUE TITLE
Path incompatibility on Windows

### ISSUE THREAD

**Issue by puglisij** [Elderjs/elderjs#112]
I'm not sure this can be run on Windows without extensive modifications to the code?
When following the Getting Started guide on windows, I'm getting the following error:
`
> Elder.js running on http://localhost:3000
/ [
  TypeError [ERR_INVALID_ARG_TYPE]: The "id" argument must be of type string. Received undefined
      at validateString (internal/validators.js:124:11)
      at Module.require (internal/modules/cjs/loader.js:945:3)
      at require (internal/modules/cjs/helpers.js:88:18)
      at Object.templateComponent (C:\Projects\elderjs-app\node_modules\@elderjs\elderjs\build\utils\svelteComponent.js:20:20)
      at buildPage (C:\Projects\elderjs-app\node_modules\@elderjs\elderjs\build\utils\Page.js:43:40)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)
      at async Object.run (C:\Projects\elderjs-app\node_modules\@elderjs\elderjs\build\hooks\index.js:81:34)
      at async C:\Projects\elderjs-app\node_modules\@elderjs\elderjs\build\utils\prepareRunHook.js:46:44
      at async processHook (C:\Projects\elderjs-app\node_modules\@elderjs\elderjs\build\utils\prepareRunHook.js:41:13)
      at async Array.prepServer (C:\Projects\elderjs-app\node_modules\@elderjs\elderjs\build\utils\prepareServer.js:8:9) {
    code: 'ERR_INVALID_ARG_TYPE'
  }
]
`

It appears this is due to the Node.js 'path' module resolving paths with **\\\\** as opposed to **/** on Unix based systems like Mac.
Long story short this results in an empty string passed to require() inside of `/build/utils/svelteComponent.ts`

The '/' pattern looks to be used extensively throughout ElderJs source.
Has anyone run into this or has an easy work-around that doesn't involve re-writing pathing in the source?

**Comment by nickreese** [comment]
@puglisij Make sure you are on the latest release. 

If you are and are still having issues, if you can create a project or better, a test that fails we can look into fixing it. 

I've spent extensive effort trying to keep windows support and even rewrote the entire test suite to cover it.

**Comment by puglisij** [comment]
Thanks @nickreese . **face palm** I was one version behind at 1.2.1. I feel stupid for not checking, but it looks like a windowsPathFix() was sprinkled around for this very thing! Nice!
Thanks for open sourcing this project too, I really like the way this is laid out so far.
